tests/sentry/digests/test_utilities.py

```diff
@@ -265,22 +186,25 @@ def test_simple(self):
         digest = build_digest(self.project, sort_records(records))[0]
 
         expected_result = {
-            self.user1.id: set(self.team1_events),
-            self.user2.id: set(self.team2_events),
-            self.user3.id: set(self.team1_events + self.team2_events),
-            self.user4.id: set(self.user4_events),
+            self.user1.actor_id: set(self.team1_events),
+            self.user2.actor_id: set(self.team2_events),
+            self.user3.actor_id: set(self.team1_events + self.team2_events),
+            self.user4.actor_id: set(self.user4_events),
         }
-        self.assert_get_personalized_digests(self.project, digest, self.user_ids, expected_result)
+
+        with self.feature("organizations:notification-all-recipients"):
+            assert_get_personalized_digests(self.project, digest, expected_result)
 
     def test_direct_email(self):
+        """When the action type is not Issue Owners, then the target actor gets a digest."""
```

Comment:
Wait so if the action type is issue owners, what happens? We never send digests to teams, right?

Reply:
When `target_type` is `ISSUE_OWNERS` it send a digest to a list of users and teams.  When `target_type` is `TEAM` it sends a digest to a team. It won't send digests to email, but teams will start getting Slack digests.

Reply:
Hmm, I wasn't aware this was the desired behavior since email digests only ever send to a user, but I suppose that makes sense given the digest settings are project based.

Reply:
Good question.  In `get_recipients_by_provider()`, we discard email settings for teams:
```py
# Teams cannot receive emails so omit EMAIL settings.
teams_by_provider = {
    provider: teams
    for provider, teams in teams_by_provider.items()
    if provider != ExternalProviders.EMAIL
}
```
so if the team is "email only" then it'll fall back to the team's `member_set`.